website/docs/r/route_table.html.markdown

```diff
@@ -85,8 +85,11 @@ attribute once the route resource is created.
 
 ## Import
 
-Route Tables can be imported using the `route table id`, e.g.
+~> **NOTE:** Importing this resource currently imports an `aws_route` resource for each route in addition to the `aws_route_table` resource. You must use `terraform state rm` to delete the imported `aws_route` resources if you plan on applying the imported configuration or Terraform will delete the actual routes. The behavior of importing `aws_route` resources with the `aws_route_table` resource will be removed in the next major version.
```

Comment:
> You must use `terraform state rm` to delete the imported `aws_route` resources if you plan on applying the imported configuration or Terraform will delete the actual routes.

This verbiage is not accurate if you add the `aws_route` configurations before applying. The distinction I made in my original ask was intentional:

> If you intend to manage routes within the `aws_route_table` resource (using the `route` argument), you must use `terraform state rm` to delete the imported `aws_route` resources or Terraform will delete the routes.

Please feel free to reconcile the difference as you see appropriate, but I think its incorrect to say "You must use `terraform state rm`" as this is not a requirement in all situations and could cause confusion.

Reply:
@bflad Even if you add the `aws_route` configuration, the actual route is still deleted.

```hcl
resource "aws_route_table" "route_table2" {
  vpc_id = "${var.vpc_id}"
}

resource "aws_route" "route1" {
  route_table_id = "${aws_route_table.route_table2.id}"
  destination_cidr_block = "${var.destination_cidr_ipv4}"
  vpc_peering_connection_id = "${var.vpc_peering_conn_2}"
}
```
Assuming the resources were previously created, if you do the following, `aws_route.route1` will be deleted.
```console
$ terraform import -input=false aws_route_table.route_table2 "${route_table2_id}"
$ terraform plan -input=false -out=newplan
$ terraform apply -input=false newplan
```